addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -483,8 +483,8 @@ QUnit.test('display channel mention suggestions on typing "#"', async function (
         public: "groups",
     });
     await this.start();
-    const composer = this.env.models['mail.composer'].create();
-    await this.createComposerComponent(composer);
+    const thread = this.env.models['mail.thread'].create({ id: 100, model: 'project.task' });
```

Comment:
`project.task` is a bad example to take in mail module, as that model does not exist here.
You can pick `res.partner` just to be safe (`mail.channel` would be acceptable too, depending on the use case, but I think it doesn't matter here)

I'm surprised though, why did you have to change this test, and not the one just after that did exactly the same thing (create composer manually without thread).

Reply:
The PR doesn't set thread required on composer actually, wasn't it the case before / shouldn't it be done?

Reply:
This PR make thread.model required (it's used inside the `_createRecordLocalId`).

Reply:
It still makes no sense why this test in particular need a change and not all the other one.

Either this diff is useless, or it's not working as expected.

Please try to make thread required on composer and see how it goes.